src/vcpkg.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "sql-odbc",
     "version-string": "1.1.0.1",
-    "dependencies": ["aws-sdk-cpp", "rapidjson", "zlib", "gtest", "curl"],
-    "builtin-baseline": "6ca56aeb457f033d344a7106cb3f9f1abf8f4e98",
-    "overrides": [
-        { "name": "aws-sdk-cpp", "version": "1.8.83#2" },
-        { "name": "rapidjson", "version": "2022-06-28#1" },
-        { "name": "zlib", "version": "1.2.12#1" },
-        { "name": "gtest", "version": "1.11.0" }
-    ]
+    "dependencies": [
+        "aws-sdk-cpp",
+        "rapidjson",
+        "zlib",
+        "gtest",
+        "curl"
+    ],
+    "builtin-baseline": "62fb5ef70474ca6d4efe6fb0887d25d3ac30b5a1"
```

Comment:
Build is failing because GH runner uses older version of vcpkg -- it does not know about this commit/baseline. Try using `a325228200d7f229f3337e612e0077f2a5307090`.

It's [this](https://github.com/microsoft/vcpkg/commit/a325228200d7f229f3337e612e0077f2a5307090) version of vcpkg. I got it from [list](https://github.com/actions/runner-images/blob/macOS-12/20230219.1/images/macos/macos-12-Readme.md) of included software. It's printed out as part of the action log
<img width="602" alt="image" src="https://user-images.githubusercontent.com/99687200/222686406-51f75e2d-9c20-4b0f-a9de-4491daa6ceee.png">